src/java/test/config/findbugsExcludeFile.xml

```diff
@@ -90,6 +90,16 @@
      <Bug code="IS"/>
   </Match>
   <Match>
+    <Class name="org.apache.zookeeper.server.quorum.Leader"/>
+    <Method name="lead"/>
```

Comment:
Following are the variables, reported for MultiThreadedCorrectness warnings.
```
QuorumPeer.tick
FastLeaderElection.logicalclock
AuthFastLeaderElection.logicalclock
```

How about using `AtomicLong & AtomicLong` instead of excluding. Please refer branch-3.5 or trunk code as a reference.

```
AtomicLong logicalclock = new AtomicLong(); /* Election instance */
AtomicInteger tick = new AtomicInteger();
```